src/applications/gi/components/AccordionItem.jsx

```diff
@@ -26,33 +27,32 @@ class AccordionItem extends React.Component {
 
   render() {
     const expanded = this.state.expanded;
+    const label = this.state.button;
     const headerClasses = classNames('accordion-button-wrapper', {
       [this.props.headerClass]: this.props.headerClass,
     });
     return (
-      <div>
-        <li>
-          <h2 aria-live="off" className={headerClasses}>
-            <button
-              onClick={this.toggle}
-              className="usa-accordion-button"
-              aria-expanded={expanded}
-              aria-controls={this.id}
-            >
-              <span className="vads-u-font-family--serif accordion-button-text">
-                {this.props.button}
-              </span>
-            </button>
-          </h2>
-          <div
-            id={this.id}
-            className="usa-accordion-content"
-            aria-hidden={!expanded}
+      <li aria-label={label}>
```

Comment:
I checked this on `localhost` and I think the `arial-label` here is not needed. The H2 + button combo will announce the group well and inform users this is an expanded accordion. Adding the aria-label to the LI is causing a repeat.